hw03-annotation/src/main/java/TestClass.java

```diff
@@ -0,0 +1,31 @@
+import annotation.After;
+import annotation.Before;
+import annotation.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestClass {
+    private static final Logger logger = LoggerFactory.getLogger(TestClass.class);
+
+    @Before
+    void setUpTest() {
+        logger.info("Before");
+    }
+
+    @Test
+    void test1() {
+    }
```

Comment:
Какой-то тест должен упасть для проверки.
```suggestion
    void test1() {
        throw new RuntimeException("fail test");
    }
```